src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -402,48 +402,20 @@
     <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.Socket.cs">
       <Link>Interop\Unix\System.Native\Interop.Socket.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.SocketEvent.cs">
+      <Link>Interop\Unix\System.Native\Interop.SocketEvent.cs</Link>
+    </Compile>
     <Compile Include="$(CommonPath)\Interop\Unix\System.Native\Interop.SocketAddress.cs">
       <Link>Interop\Unix\System.Native\Interop.SocketAddress.cs</Link>
     </Compile>
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetsLinux)' == 'true' ">
-    <Compile Include="System\Net\Sockets\SocketAsyncEngineBackend.Linux.cs" />
     <Compile Include="System\Net\Sockets\SocketPal.Linux.cs" />
-
-    <Compile Include="$(CommonPath)\Interop\Linux\Interop.Libraries.cs">
-      <Link>Interop\Linux\Interop.Libraries.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\Interop\Linux\libc\Interop.epoll_create1.cs">
-      <Link>Interop\Linux\libc\Interop.epoll_create1.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\Interop\Linux\libc\Interop.epoll_ctl.cs">
-      <Link>Interop\Linux\libc\Interop.epoll_ctl.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\Interop\Linux\libc\Interop.epoll_event.cs">
-      <Link>Interop\Linux\libc\Interop.epoll_event.cs</Link>
-    </Compile>
-    <Compile Include="$(CommonPath)\Interop\Linux\libc\Interop.epoll_wait.cs">
-      <Link>Interop\Linux\libc\Interop.epoll_wait.cs</Link>
-    </Compile>
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetsOSX)' == 'true' ">
-    <Compile Include="System\Net\Sockets\SocketAsyncEngineBackend.OSX.cs" />
```

Comment:
Looks like the only thing standing in the way of recombining the Linux and OSX implementations is this one ```SocketPal.*.cs```... do we think we'll be able to recombine that as well, or do we expect we'll still have Linux vs OS X managed binaries?